sections/faqs/extend-and-styled-difference.js

```diff
@@ -0,0 +1,25 @@
+import md from 'components/md'
+
+const ExtendAndStyled = () => md`
+  ## What is the difference between \`Comp.extend\` and \`styled(Comp)\`?
+
+  ### Behind the scenes
+  The \`styled()\` factory generates new component styles with a new class. The classnames are
+  then passed to the React component via the \`className\` prop.
+  Calling \`extend\` creates new component styles by extending the old one, and thus doesn't
+  generate two classes for a single component. (\`styled()\` factory does that)
+
+  ### Which one should you use
+  We recommend to use \`.extend\` where possible for internal \`StyledComponents\`.
+  Try to use the \`styled()\` factory however for external or cross-file components
+  where it isn't clear what it's going to be when writing the wrapper \`StyledComponent\`.
+
+  Keep in mind that wrapping your own container and structural components is not recommended.
+  It should always be attempted to wrap presentational components only since
+  cleaner logic can be achieved when the structural abstracts and encapsulates the presentational logic.
```

Comment:
Maybe for clarity, since you said it's not easy to understand:

> Keep in mind that wrapping your own components with `styled()` is not recommended.
> It should always be attempted to only have `StyledComponents` and not wrap any wrappers and containers at a higher level. Otherwise your styling logic will be scattered across multiple files, and it will become tempting to use child selectors to target elements inside your containers. This of course increases the specificity of rules we generate, which can lead to more code and `styled()` wrappers which try to compensate for this up in your component hierarchy. Instead, you can keep your UI code clean by bundling your styling at the lowest level, and utilise themes, inheritance, mixins, props, and even "reverse selectors".

Reply:
@philpl Why don't you explain anything related to `extend`?

Reply:
You know, I think we're explaining too much here. Although, if we want to have all this, you should divide it into more 2 or 3 paragraphs. I need to get a quick answer after reading a few lines, then if I wanted to know more, I can continue. Does it make sense to you?

Reply:
Ah, it's just for the last paragraph?

Reply:
> since you said it's not easy to understand:

In fact, I didn't say that hahaha I just copy pasted what you said earlier.